syntaxes/styled-components.json

```diff
@@ -41,7 +41,7 @@
           "name": "punctuation.definition.string.template.begin.js string.template.js"
         }
       },
-      "end": "(`)(\\))?",
+      "end": "`",
```

Comment:
If you change this back the end captures don't make any sense any more. It expects `1` and `2`, but without the braces you'll only have `0`. So maybe change it like
```
        "0": {
          "name": "punctuation.definition.string.template.end.js string.template.js"
        },
```

and remove the 2. This could also result in the fixture output to be fine 👍

Reply:
Made that change, and indeed all the fixtures when back to the way they were before. 

I don't 100% understand how the plugin works, so I was kind of just giving it my best guess!

Reply:
@adbutterfield are you still working on this?